client/src/components/player.js

```diff
@@ -16,6 +16,17 @@ function GetRaceIcon(race) {
 }
 
 function Player(player) {
+  let name = <Typography variant="h6">{player.name}</Typography>;
+
+  if(player.alias) {
```

Comment:
Change this to be: `if (player.alias && player.name != player.alias) {` to avoid things like: "NightEnD as NightEnD"